src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -261,22 +264,18 @@ public ReadTransformer makePostReadFilterTransformer(){
     }
 
     /**
-     * @return Default size in MB of the cloud prefetch buffer. May be overridden by individual tools.
-     *         The default implementation returns a value (40 MB) that is suitable for tools with a small
-     *         number of large cloud inputs. Tools with large numbers of cloud inputs will likely want to
-     *         override to specify a smaller size.
+     * @return The cloud prefetch buffer size as set in {@link GATKConfig}
      */
-    public int getDefaultCloudPrefetchBufferSize() {
-        return 40;
+    public int getCloudPrefetchBufferSize() {
+        return ConfigCache.getOrCreate(GATKConfig.class).cloudPrefetchBuffer();
```

Comment:
I guess that this should be `ConfigUtils` instead of `ConfigCache`...

Reply:
Yup.  Fixed!